src/Microsoft.DotNet.Migration/DefaultMigrationRuleSet.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Cli;
+using System.Linq;
+using System.IO;
+using Newtonsoft.Json;
+using Microsoft.DotNet.Migration.Rules;
+
+namespace Microsoft.DotNet.Migration
```

Comment:
Is this too general of a name for the assembly/package/namespace?  I'm sure there will be other "migrations" across .NET.

Maybe Microsoft.DotNet.ProjectModel.Migration?  Or something that connects it with "project.json".